src/main/java/org/example/service/DeveloperService.java

```diff
@@ -48,9 +48,9 @@ public List<Developer> getDevelopers() {
 
                 developers.add(developer);
             }
-            LOGGER.info("Successful developers execution!");
+            LOGGER.info("Successful {} developers execution!", developers.size());
```

Comment:
Good, check this book and write correct sentence:
https://eclass.teiwm.gr/modules/document/file.php/DNG225/Grammarway_4.pdf

Reply:
sentence is correct

Reply:
fixed sentence

Reply:
Still wrong

Reply:
👽